src/toil/server/api_spec/workflow_execution_service.swagger.yaml

```diff
@@ -0,0 +1,651 @@
+basePath: '/ga4gh/wes/v1'
+swagger: '2.0'
+info:
+  title: Workflow Execution Service
```

Comment:
Does this file have an applicable license that we should note in here or in Toil's LICENSE file? Or in a LICENSE in the enclosing directory?

Reply:
Would it be sufficient to add a license header to the file, and include their full license in the enclosing directory? 

https://github.com/ga4gh/workflow-execution-service-schemas/blob/develop/LICENSE

Reply:
If the code is under a license that requires shipping the license file, I'm pretty sure we do need to ship the license file. I don't think we need to add a license header if it didn't come with a license header to start with, but I think we're allowed to.